virtfs-ach/src/tests/writer.rs

```diff
@@ -1,22 +1,26 @@
 use std::io::{BufReader, Cursor, Seek, SeekFrom};
 use crate::{loader::Archive, writer::ArchiveBuilder};
 
-const FILE_1: &[u8] = include_bytes!("../../spec/main.txt");
-const FILE_2: &[u8] = include_bytes!("../../Cargo.lock");
+// this is complete gibberish
+// generated with "dd if=/dev/urandom of=data.bin bs=1 count=1000000"
+const FILE_1: &[u8] = include_bytes!("../../test-data/data.bin");
+const FILE_1_PATH: &str = "data.bin";
+const FILE_2: &[u8] = include_bytes!("../../test-data/lorem_ipsum.txt");
+const FILE_2_PATH: &str = "lorem_ipsum.txt";
 
 #[test]
 fn archive_writing() {
     let mut builder = ArchiveBuilder::new();
     builder.add_file(
         FILE_1.to_owned(),
```

Comment:
Maybe just dereference here

Reply:
rust won't let me dereference that without setting the exact size, which is really impractical